nav2_lifecycle_manager/include/nav2_lifecycle_manager/lifecycle_manager.hpp

```diff
@@ -38,6 +38,17 @@ namespace nav2_lifecycle_manager
 using namespace std::chrono_literals;  // NOLINT
 
 using nav2_msgs::srv::ManageLifecycleNodes;
+
+/// @brief Enum to for keeping track of the state of managed nodes
+enum NodeState
+{
+  UNCONFIGURED,
```

Comment:
The 4 primary states are unconfigured, inactive, active, and finalized. I don't know what "paused" is

Reply:
Indeed it is, change made!